src/components/common/DetailComment.vue

```diff
@@ -100,6 +115,12 @@ export default class DetailComment extends Vue {
       padding-top: 0.5rem;
       padding-left: 0.5rem;
     }
+
+    .theme--light.v-label {
```

Comment:
I get why you're doing this but I object to use "theme light" as a  selector. It would be better to use this (parent then child), which don't depend on theme:
```
.v-text-field__slot .v-label
```